src/axolotl/utils/models.py

```diff
@@ -1322,7 +1322,7 @@ def load_lora(model, cfg, inference=False, config_only=False):
         target_modules=lora_target_modules,
         layers_to_transform=cfg.peft_layers_to_transform,
         layers_pattern=cfg.peft_layers_pattern,
-        lora_dropout=cfg.lora_dropout,
+        lora_dropout=cfg.lora_dropout if cfg.lora_dropout is not None else 0.0,
```

Comment:
Could we perhaps move this higher up in stack? Maybe to the validation stage instead? I think it's better to centralize this.